standard/standard_3.php

```diff
@@ -739,6 +739,9 @@ function fmod(float $num1, float $num2): float
  * Performs a floating-point division under
  * IEEE 754 semantics. Division by zero is considered well-defined and
  * will return one of Inf, -Inf or NaN.
+ * @param float $num1
```

Comment:
Empty `@param`s tag can be avoided in case there is a parameter type in the signature.

Reply:
Should I revert it?